site/_posts/2017-03-20-yet-another-move-by-euwe.md

```diff
@@ -0,0 +1,94 @@
+
+[Adam](https://github.com/agrare) moved [fog-vcloud-director gem](https://github.com/ManageIQ/manageiq/pull/14358) to [manageiq-providers-vmware gemspec](https://github.com/ManageIQ/manageiq-providers-vmware/pull/28)
+
+[James](https://github.com/jameswnl) moved [Credential's API_ATTRIBUTES to the shared one](https://github.com/ManageIQ/manageiq/pull/14334)
+
+## Docs
+[Ladas](https://github.com/lsmola) is showing to us that developer can also create nice documentation.
+Let's inspire and look on his [documentation for InventoryCollection.new](https://github.com/ManageIQ/manageiq/pull/14346) - examples included!
+
+## Fixed
+Time is most important physical quantity on the world and also [Keenan](https://github.com/kbrock) is aware of this fact. It is visible in his [peformance PR](https://github.com/ManageIQ/manageiq/pull/14285 "second part") to speed up for generating widgets.
+
+[Beni](https://github.com/cben) with his [Container TLS fixes](https://github.com/ManageIQ/manageiq-ui-classic/pull/670) is fixing ability to create and edit container's provider without Hawkular.
+
+Broken chargeback has been fixed by [Allen](https://github.com/ManageIQ/manageiq-ui-service/pull/591) in [ManageIQ UI Service][manageiq-ui-service] repo.
+
+## New
+[Hungarian](https://github.com/skateman), [Czech](https://github.com/martinpovolny) and [Ame](https://github.com/abellotti)[ric](https://github.com/Fryguy)[ans](https://github.com/yrudman) are creating a pull request ... it is not begining of joke, they worked together on one PR, concretely they added [option for disabling asynchronous](https://github.com/ManageIQ/manageiq/pull/13898) notifications.
+
+API is still rapidly improved by  [Jillian](https://github.com/jntullo). Her contribution is almost [tenth of PRs](https://github.com/ManageIQ/manageiq/pulls?q=is%3Apr+is%3Amerged+base%3Amaster+merged%3A%222017-03-13+..+2017-03-19%22+sort%3Acreated-desc+author%3Ajntullo) in ManageIQ repo!
+
+Sharing is good and you mother was right as well as [Richard](https://github.com/rwsu) in his [PR](https://github.com/ManageIQ/manageiq/pull/13535) where he introduced CloudTenancyMixin - it allows to us share tenant functionality also for Cloud Objects. 
+
+[Daniel](https://github.com/djberg96) modified label_tag_mapping_add to accommodate scoped entity type in his [PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/666)
+*Warning!!! The PR is not for people with hexakosioihexekontahexaphobia*
+
+[Erez](https://github.com/ManageIQ/manageiq/pull/13704) added [External Logging Link](https://github.com/ManageIQ/manageiq/pull/13704) for container nodes.
+
+
+## Deleted
+
+Best code is no code. Our contributors are coding hand by hand with this statement, see their deletions: 
+
+- [James](https://github.com/jameswnl) - [Remove uncalled miq_server lets](https://github.com/ManageIQ/manageiq/pull/14383): `+1,181 −16,115`
```

Comment:
James' PR link seems to have the same description as the one below but I don't think that's right?